src/test/java/edu/ucla/library/iiif/auth/verticles/MainVerticleIT.java

```diff
@@ -58,6 +68,55 @@ public void testThatTheServerIsStarted(final Vertx aVertx, final VertxTestContex
         });
     }
 
+    /**
+     * Tests that a client can get an item's access level.
+     *
+     * @param aVertx A Vert.x instance
+     * @param aContext A test context
+     */
+    @Test
+    public void testGetAccessLevel(final Vertx aVertx, final VertxTestContext aContext) {
```

Comment:
Why not put this in an access level handler test? That seems more specific than MainVerticleIT. Or do we need handler tests in addition to this?

Reply:
I have not been able to find a good method for "unit testing" the `handle` method of a Handler; the easiest way to test it seems to be hooking it up to a HTTP server and calling the server with a WebClient (as is being done here). The reason I made a unit test file for AccessCookieHandler is because it has an additional method `isOnNetwork`.

Reply:
Yes, I agree... we can swap out whatever the handler talks to but communicating with it seems to require going through the web server. On reason I still like putting that in a handler test though is one can scan the code and look for tests that correspond to a particular handler. When we have multiple handlers, if we put them all in the main verticle, we then have to scan that file to see which are being tested.

Reply:
Oh, I was reading your comment literally as if you were asking why it wasn't in "AccessLevelHandlerTest". I do like the idea of spitting that file out into separate handler integration test classes.